docs/spec_draft.md

```diff
@@ -761,6 +762,39 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.count_leading_zeros
+
+### Semantics
+
+Performs element-wise count of the number of leading zeros in the `operand`
+tensor and produces a `result` tensor.
+
+### Inputs
+
+| Name      | Type                   |
+|-----------|------------------------|
+| `operand` | tensor of integer type |
+
+### Outputs
+
+| Name     | Type                   |
+|----------|------------------------|
+| `result` | tensor of integer type |
+
+### Constraints
+
+  * (C1) `operand` and `result` have the same type.
+
+### Examples
+
+```mlir
+// %operand: [[0, 1], [127, -1]]
+%result = "stablehlo.count_leading_zeros"(%operand) : (2x2xi8) -> tensor<2x2xi8>
```

Comment:
This is not a valid MLIR syntax. Fixed: `%result = "stablehlo.count_leading_zeros"(%operand) : (tensor<2x2xi8>) -> tensor<2x2xi8>`.

Reply:
Thanks for catching this!